src/organisms/cards/PlaybackCard/playback_card.module.scss

```diff
@@ -0,0 +1,22 @@
+@import 'assets/stylesheets/global';
+
+$playback-card-border: 1px solid color('neutral-5');
+
+.playbackCard {
+  border-radius: 6px;
+  border: $playback-card-border;
```

Comment:
see variable comments above, I don't think this is necessary since it's used once